ArtifactScript.py

```diff
@@ -57,4 +96,5 @@ def GetCardAbility(card):
             if x["card_id"] == card["references"][0]["card_id"]:
                 print("Signature Card:" + " " + x.get("card_name")["english"])
 
+GetAPIData()
```

Comment:
Currently looks like this function is downloading both card sets twice. And then `GetAPIData()` gets called again.

Reply:
Actually, this might be fine. I was looking at the call on line 76 and thought there was a potential bug with `APIExpire`.

I'm not familiar enough with global variables in python. I was worried that the declaration of `Expiretime` on line 22 would shadow the one on line 14, but after a quick google search that does not seem to be a problem.